sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -596,39 +613,61 @@ def __init__(self, **kwargs):
         self.submodels = kwargs.get("submodels", None)
         self.errors = kwargs.get("errors", None)
         self.training_documents = kwargs.get("training_documents", None)
+        self.display_name = kwargs.get("display_name", None)
+        self.properties = kwargs.get("properties", None)
 
     @classmethod
-    def _from_generated(cls, model):
+    def _from_generated(cls, model, api_version):
         return cls(
             model_id=model.model_info.model_id,
             status=model.model_info.status,
             training_started_on=model.model_info.created_date_time,
             training_completed_on=model.model_info.last_updated_date_time,
             submodels=CustomFormSubmodel._from_generated_unlabeled(model)
-            if model.keys else CustomFormSubmodel._from_generated_labeled(model),
+            if model.keys else CustomFormSubmodel._from_generated_labeled(
+                model, api_version, display_name=model.model_info.model_name
+            ),
             errors=FormRecognizerError._from_generated(model.train_result.errors)
             if model.train_result else None,
             training_documents=TrainingDocumentInfo._from_generated(model.train_result)
-            if model.train_result else None
+            if model.train_result else None,
+            properties=CustomFormModelProperties._from_generated(model.model_info),
+            display_name=model.model_info.model_name
+        )
+
+    @classmethod
+    def _from_generated_composed(cls, model):
+        return cls(
+            model_id=model.model_info.model_id,
```

Comment:
are we intentionally hiding errors here?

Reply:
Not sure what you mean by hiding. We decided that a composed model would not return a list of errors, so I am not passing it into the ctor. It will get set to `None` in this case.

Reply:
ohh that makes sense. Thank you!